cubeseed/address/tests.py

```diff
@@ -0,0 +1,81 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from rest_framework.reverse import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from .models import Address
+
+
+class AddressModelTest(TestCase):
+    def test_ng_address_1(self):
+        addr = Address()
+        addr.address = "42, Olowu Street"
+        addr.save()
+        self.assertEqual(addr.osm_checked, True, msg="OSM Check failed")
+        self.assertAlmostEqual(addr.osm_latitude, 6.5963104, msg="wrong latitude")
```

Comment:
Tests passed when the latitude and logitude are changed to the value to 6.5990707, 3.3430275 may be we should keep that

Reply:
I didn't understand this.

Reply:
The tests were failing because of the coordinates; they were not exactly Nigeria's location(6.5990707, 3.3430275) so changing that to the respective latitude and longitude passed the tests; I assume we are using Nigeria as a demo location for now right?

Reply:
I'm still lost, I put the coordinates returned by osm on the test and the test was passing
Nigeria is the demo location and the only location for now

Reply:
these are the returned locations when i tested it